File: neuroguard-backend/medical-history-service/src/main/java/com/neuroguard/medicalhistoryservice/dto/UserDto.java
package com.neuroguard.medicalhistoryservice.dto;


import lombok.Data;

@Data
public class UserDto {
    private Long id;
    private String username;
    private String email;
    private String firstName;
    private String lastName;
    private String role;  // ADMIN, PATIENT, PROVIDER, CAREGIVER
    private String gender; //
    private Integer age;
}

File: FrontEnd/src/app/core/models/user.dto.ts
export interface UserDto {
  id: number;
  username: string;
  email: string;
  firstName: string;
  lastName: string;
  role: string;
  connected?: boolean;
  status?: 'ACTIVE' | 'BANNED' | 'DISABLED';
  bannedUntil?: string; // ISO datetime string
  longitude?: number;
  latitude?: number;
  altitude?: number;
  gender?: string;
  age?: number;
}